### PR TITLE
Allfeatures

### PR DESCRIPTION
**Pull request by BenMKT**
- Created styles.css file and added all basic CSS styles.
- Fixed all linter errors. All checks passing.
- Created HTML content and added it to the skeleton index.html file.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  width: 1.25em;
+  height: 1.19em;
+  margin-right: 0.625em;
+}
+
+.menu img:hover {
+  cursor: pointer;
+}
+
+@media screen and (min-width: 768px) {
```
Comment (braincee) [inline styles.css:56]: Good job implementing the minimum desktop size. 👍

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  background-image: url(img/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-size: auto;
+  padding: 2em;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (braincee) [inline styles.css:12]: Good job implementing flexbox in the header section. 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,93 @@
...
+Run git clone [https://github.com/BenMKT/Portfolio]
+
+### Prerequisites
+
+Basic knowledge of HTML, CSS, Git and GitHub.
+
+### Setup
+
+Clone this repository to your desired folder.
+
```
Comment (braincee) [inline README.md:48]: - [ ] Please ensure adding specific setup command for developers to clone your repository.
Example: 
```
cd my-folder
git clone https://github.com/BenMKT/Portfolio.git
cd tab enter
```

File: README.md [hunk 4]
```diff
@@ -1 +1,93 @@
...
+<ol>
+  <li>Code Editor (Visual Studio Code,Atom or Sublime)</li>
+  <li>Git (GitBash)</li>
+</ol>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
```
Comment (braincee) [inline README.md:59]: - [ ] Please ensure adding social other social media handles to the author section which makes your repository professional for recruiters to hire you. Example `LinkedIn` and `Twitter`

File: README.md (hunk 3 above)
Comment (braincee) [inline README.md:48]: - [ ] Please ensure adding specific setup command for developers to clone your repository. 😄 
Example:
```
cd my-folder
git clone https://github.com/BenMKT/Portfolio.git
cd tab enter
```

File: README.md [hunk 5]
```diff
@@ -1 +1,93 @@
...
+### Setup
+
+Clone this repository to your desired folder.
+
+### Install
+
+<ol>
+  <li>Code Editor (Visual Studio Code,Atom or Sublime)</li>
+  <li>Git (GitBash)</li>
+</ol>
```
Comment (braincee) [inline README.md:54]: - [ ] Please ensure adding a specific installation command. Example: `npm install / npm i`. 😃

File: README.md [hunk 6]
```diff
@@ -1 +1,93 @@
...
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@BenMKT](https://github.com/BenMKT/Portfolio)
```
Comment (braincee) [inline README.md:61]: - [ ] Please ensure adding social other social media handles to the author section which makes your repository professional for recruiters to hire you. 😃 
- [ ] Example `LinkedIn` and `Twitter`

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
+body {
```
Comment (braincee) [inline styles.css:1]: - [ ] Please ensure you make your project look like the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

**_REQUIRED FIGMA TEMPLATE DESIGN_**

Screenshot (73)


_**YOUR DESIGN**_

Screenshot (77)

File: README.md [hunk 8]
```diff
@@ -1 +1,93 @@
...
+- [ ] **Dynamic webpage through Javascript adoption.**
+- [ ] **Webpage responsiveness on generally all devices.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (braincee) [inline README.md:75]: - [ ] [OPTIONAL] I kindly suggest adding issues link to the README file. This makes your README file looks professional and other developers can create issues by clicking on the link provided. 😄

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @BenMKT ✋, I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------

- [ ] Please I suggest adding flexbox to the `main` content as it is part of the project requirements. 😃 
- [ ] Please I suggest adding a minimum mobile size of `375px` as it is part of the project requirements. 😃 
- [ ] Please I suggest adding responsive values in your CSS rules (i.e. `width: 50%`) as it is part of the project requirements. 😄 

**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

- [ ] [OPTIONAL] Please I suggest adding imperative tenses to your commit messages.
Example: `Add ...` not `Added ...`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (BenMKT) [reply README.md]: Rectified as per review

File: README.md (hunk 6 above)
Comment (BenMKT) [reply README.md:69]: Add Linkedin to author

File: README.md (hunk 4 above)
Comment (BenMKT) [reply README.md:67]: LinkedIn now included

File: README.md (hunk 3 above)
Comment (BenMKT) [reply README.md]: Now updated on both install and setup headings

File: styles.css (hunk 7 above)
Comment (BenMKT) [reply styles.css:1]: Add margin in HTML container for spacing

File: index.html [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    <title>Benson Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="header">
+    <div class="logo">My Logo</div>
+    <div class="menu">
+        <img src="img/Normal Button/Tertiary/Icons/Union.png" alt=""> 
+    </div>
```
Comment (Shedrack-Sunday) [inline index.html:15]: - [Optional] Though missed by the previous reviewer a major requirement, kindly wraps this section in a `header` tag and the hamburger in a `nav` bar. These changes cause a proper semantic flow for your website which helps in web accessibility

File: index.html [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+  </div>
+  <div class="footer">
+    <p>LET’S CONNECT</p>
+    <div class="icons">
+        <a href=""><img src="img/Vector.png"  alt="twitter"></a>
+        <a href=""><img src="img/Linkedin icon.png" alt="linkedin"></a>
+        <a href=""><img src="img/Vector (3).png" alt="mail"></a>
+        <a href=""><img src="img/Vector (2).png" alt="github"></a>
+        <a href=""><img src="img/Vector (4).png" alt="site"></a>
```
Comment (Shedrack-Sunday) [inline index.html:30]: - [Optional] Also missed by the previous, The list of icons should be a `ul` tag to group them, and `li` per icon; each icon has a link (a tag).

- Apart from being semantically correct, the tag will let you connect your social networks with the icons, using href.

File: README.md [hunk 11]
```diff
@@ -1 +1,103 @@
...
+</ol>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@BenMKT](https://github.com/BenMKT/Portfolio)
+- linkedin: [@BenK](https://linkedin)
+
```
Comment (Shedrack-Sunday) [inline README.md:72]: - [ ] Though you made the change by the previous reviewer, the link is an invalid one. It is good practice to always pay attention to details when adding clickable links. the right LinkedIn link is `www.linkedin/in/<your user name>`

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @BenMKT     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights:
✔️ README setup
✔️  Linters passed
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- A few code related  issues need to be fixed to get your code perfect, and you already have a nice implementaion 👏

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 10 above)
Comment (BenMKT) [reply index.html]: - Suggestions incorporated although optional. Some of the links I dont have such accounts. I have inserted links for those that I have accounts to.
- Had to include the icons as it is a requirement in the project. Will omit them in the future past this stage of the project.

File: README.md (hunk 11 above)
Comment (BenMKT) [reply README.md]: - inserted the correct LinkedIn link in the readme author title.

File: index.html (hunk 9 above)
Comment (BenMKT) [reply index.html]: -  Added the semantic tags as per review for web accessibility though optional.

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @BenMKT 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: no linter errors 
:+1: design

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**